lib/DDG/Spice/Metar.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::Metar;
+
+# Metar: Looks up the METAR information for a given airport. 
+# The input should be a 4 letter ICAO code.
+# It recognizes the query "metar ICAO" and "ICAO metar". 
+
+use DDG::Spice;
+
+spice to => 'http://avwx.rest/api/metar.php?station=$1&format=JSON';
+spice proxy_cache_valid => "418 1d"; # Do not cache responses.
+spice wrap_jsonp_callback => 1;
+
+triggers any => 'metar';
+
+handle words => sub {
```

Comment:
I have now added the validation using Geo::ICAO. (And I maintained the original logic of checking each of the two words.)